helm/ph-ee-fin-back-end/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: back-end
+description: A Helm chart for front-end services
+type: application
+version: 1.0.0
+appVersion: "ph-ee-superset: v1.16.0; ph-ee-engine:v1.16.0; fin-engine: v1.0.0; operations: 1.0.0;"
+
+dependencies:
+- name: ph-ee-superset
+  repository: "https://fynarfin.io/images/superset/"
+  version: 0.2.0
```

Comment:
We need to release a version of Superset before publishing this chart. (0.*.* is usually not considered as a public release]